<launch>
	<group ns="motion_control">
		<!-- TODO: Add params -->
		<node pkg="motion_control" type="inverse_kinematics.py" name="inverse_kinematics"/>
		<node pkg="motion_control" type="wheel_speed_controller.py" name="wheel_speed_controller"/>
	</group>

</launch>